src/common/config_opts.h

```diff
@@ -13,7 +13,7 @@
  */
 
 /* note: no header guard */
-OPTION(host, OPT_STR, "localhost")
+OPTION(host, OPT_STR, "default")  // "default" means that host will be `hostname -s` 
```

Comment:
OK! sounds perfect, I am updating it. 

so, we don't need to provide ${hostname} and ${fdqn} now?